pkg/controller/vm_controller.go

```diff
@@ -323,6 +323,22 @@ func (r *VMReconciler) buildVMPod(ctx context.Context, vm *virtv1alpha1.VirtualM
 		})
 	}
 
+	if vm.Spec.Instance.Memory.Hugepages != nil {
+		vmPod.Spec.Volumes = append(vmPod.Spec.Volumes, corev1.Volume{
+			Name: "hugepages",
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{
+					Medium: "HugePages",
```

Comment:
What does this medium field mean?

Reply:
A pod may use medium: HugePages only if it requests huge pages of one size.
See more details: https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/

Reply:
According to [Kubernetes API ref](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#emptydirvolumesource-v1-core), medium must be an empty string or Memory. Is HugePages a valid value here?